### DISCUSSION TITLE
Accessibility improvements to navigation

### DISCUSSION THREAD

**Discussion by benjiallen** [community/community#55172] [Accessibility]
### Select Topic Area

Show & Tell

### Body

GitHub have recently introduced some improvements to the primary navigation of github.com. It's a recent release and we're still fixing bugs, but it's in a good place to share and gather feedback.

You can activate this feature using [feature preview](https://docs.github.com/en/get-started/using-github/exploring-early-access-releases-with-feature-preview). If you’re finding feature preview difficult to use, please add a comment to this discussion and the accessibility team will add you manually.

You can view a [demo of keyboard accessibility within the new navigation on YouTube](https://www.youtube.com/watch?v=Fvvg1xwMyb0&list=PL0lo9MOBetEGvxp03Kh3eIlFvIvfE21KF&index=6).

Thanks, looking forward to the feedback 👍

**Comment by nandgator** [comment]
| New | Old |
| --- | --- |
| ![image](https://github.com/community/community/assets/38415384/484469f1-162b-4703-874c-3162509c0854) | ![image](https://github.com/community/community/assets/38415384/c3690b39-107b-4658-a6ab-58c750864cef) |

I kind of love-hate the new global/primary navigation.

Main thing that bugs me, in the new navigation design, is **lack of** ability to `watch` / `star` / `fork` / `download` zipped repository from anywhere but the `<> Code` page / section.
